TravelExpertsGUI/frmMain.cs

```diff
@@ -1,26 +1,57 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.ComponentModel;
 using System.Data;
-using System.Drawing;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
+using TravelExpertsData;
+using TravelExpertsDataAPI;
 
 namespace TravelExpertsGUI
 {
     public partial class frmMain : Form
     {
+        // productKeyMap maps the ListBox indices to the ProductId from the database
+        Dictionary<int, int> productKeyMap = new Dictionary<int, int>();
+
+        // A list of all products in the database
+        List<Product> products;
```

Comment:
Name `products` is to ambiguous. Please change to `productsTabProducts` for better clarity